<launch>
  <!-- Launch file for velodyne16 VLP-16 LiDAR -->

    <arg name="rviz" default="true" />

    <rosparam command="load" file="$(find lio_sam)/config/velodyne64.yaml" />


    <node pkg="lio_sam" type="fastlio_mapping" name="laserMapping" output="screen" > 
      <param name="feature_extract_enable" type="bool" value="0"/>
      <param name="point_filter_num" type="int" value="4"/>
      <param name="max_iteration" type="int" value="3" />
      <param name="filter_size_surf" type="double" value="0.5" />
      <param name="filter_size_map" type="double" value="0.5" />
      <param name="cube_side_length" type="double" value="1000" />
      <param name="/robot_id" type="string" value = "jackal1"/>
     </node>    
      
    <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find lio_sam)/launch/include/config/loam_livox.rviz" />
    </group>

</launch>
